lib/stardate.rb

```diff
@@ -13,6 +13,10 @@ def to_stardate
       seconds = self - t2          # seconds passed since the beginning of the day
       ((self.year - YEAR_O) + days / YEAR_DURATION + seconds / (YEAR_DURATION * 24 * 3600)) * 1000
     end
+    
+    def to_captains_log
+      "Captain's log, stardate #{to_stardate}..."
```

Comment:
Certainly Wesley! That syntax allows us to include the value of a variable within a string. We refer to it as "string interpolation".